database/migrations/2021_03_01_151400_create_keyword_search_locale_table.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateKeywordSearchLocaleTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('keyword_search_locale', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(app('keyword'))->index();
+            $table->foreignIdFor(app('search_locale'))->index();
+            $table->timestamps();
```

Comment:
Let's add creator & updater to this so we know who added them.

Reply:
👍